oat-sa/extension-tao-itemqti
# Feat/tr 917 rtl match interaction

Related to https://oat-sa.atlassian.net/browse/TR-917
Composer file https://github.com/oat-sa/kitchen-playground/blob/mca/tr8x/files/delivery/composer.json
PHP 7.2, Composer 2

I have to delete dynamic style in  views/js/qtiCreator/widgets/interactions/matchInteraction/states/Question.js:85
could you check if you can find any problems with it so far it looks good.
clean up a bit of duplication of styles

To test
1) Install using composer above
2) go to authoring
3) create an item with match interaction better to check choices interaction as well
4) also please check response mode for interaction.
5) and items controls in the frame around the item (blue header and button at bottom)

All should be good in LTR and accordingly in RTL mode
![preview](https://user-images.githubusercontent.com/2067027/134007016-f6d8a3a8-b434-4fb4-ae6b-07d2d36ecbee.png)
